rules.txt

```diff
@@ -329,17 +329,28 @@
 /do-?something-?big                       301  /2024-02-27-do-something-big-explore-a-career-with-the-city-of-philadelphia/
 /dobbins                                  301  /programs/community-schools/murrell-dobbins-cte-high-school/
 /documents/2015-income-based-wage-tax-refund-petition 301 /documents/income-based-wage-tax-refund-petition/
+/documents/2021-(.*)                      301  /departments/department-of-revenue/forms-documents/tax-forms-and-instructions/#/
```

Comment:
I know it's tempting to use regexes here but I think we need to only redirect the specific pages, because any other department can create a document page that begins with 2021- or 2022- etc